draft-ietf-webtrans-http2.md

```diff
@@ -524,6 +564,12 @@ field. In-order delivery of WT_STREAM capsules ensures that the amount of
 session-level flow control consumed by a stream is always known by both
 endpoints.
 
+A WT_RESET_STREAM capsule MUST NOT be sent after a stream is closed or reset.
```

Comment:
Should we add a note about intermediaries? A naive intermediary that converts QUIC RESET_STREAM to WT_RESET_STREAM is going to have a bad day, so maybe we need a "don't do that" sign

Reply:
Good call.  I added a general note as this relates to multiple frame/capsule types.